src/category_theory/endofunctor/algebra.lean

```diff
@@ -394,22 +390,25 @@ def coalgebra.to_algebra_of (adj : F ⊣ G) : coalgebra G ⥤ algebra F :=
 adjoint and going back is isomorphic to the identity functor. -/
 def alg_coalg_equiv.unit_iso (adj : F ⊣ G) :
   𝟭 (algebra F) ≅ (algebra.to_coalgebra_of adj) ⋙ (coalgebra.to_algebra_of adj) :=
-{ hom := { app := λ A, { f := (𝟙 A.1),
-                           h' := begin dsimp, rw [F.map_id, category.id_comp, category.comp_id],
-                                      apply (adj.hom_equiv _ _).left_inv A.str, end },
-                         naturality' := λ A₁ A₂ f,
-                           begin dsimp, ext1, dsimp, rw[category.id_comp, category.comp_id],
-                                 refl end, },
-                inv := { app := λ A, { f := (𝟙 A.1),
-                          h' := begin dsimp, erw [F.map_id, category.id_comp, category.comp_id],
-                                      apply ((adj.hom_equiv _ _).left_inv A.str).symm end },
-                         naturality' := λ A₁ A₂ f,
-                           begin dsimp, ext1, dsimp, erw [category.comp_id, category.id_comp],
-                                 refl, end },
-                hom_inv_id' := begin dsimp, ext1, dsimp, ext1, dsimp, ext1, dsimp,
+{ hom :=
```

Comment:
I laid it out a bit more legibly, I hope this is the right way of doing things - probably would be best to consult another person as well on this